cluster-scope/base/apps/daemonsets/kepler-exporter/daemonset.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kepler-exporter
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: exporter
+      app.kubernetes.io/name: kepler-exporter
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: exporter
+        app.kubernetes.io/name: kepler-exporter
+    spec:
+      hostNetwork: true
+      serviceAccountName: kepler-sa
+      containers:
+      - name: kepler-exporter
+        image: quay.io/sustainable_computing_io/kepler:operate-first
+        imagePullPolicy: Always
+        requests:
+          memory: "400MiB"
+          cpu: "100m"
```

@HumairAK 
In this case there seems to be legitimate issue with manifests 
> WARN - stdin contains an invalid DaemonSet (kepler-monitoring.kepler-exporter) - requests: Additional property requests is not allowed

From looking at the [documentation](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) I think there should be `containers.resources.requests` and not `containers.requests`
```suggestion
      containers:
      - name: kepler-exporter
        image: quay.io/sustainable_computing_io/kepler:operate-first
        imagePullPolicy: Always
        resources:
          requests:
            memory: "400MiB"
            cpu: "100m"
```

---

copy paste error. fixed.